### DISCUSSION TITLE
Welcome to ktmidi Discussions!

### DISCUSSION THREAD

**Discussion by atsushieno** [atsushieno/ktmidi#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Burtan** [comment]
Hi,
I'm frederik. I'm a keyboardist who builds some apps for himself and if they are stable, distributes them. I'm currently using ktmidi on a kotlin multiplatform project to manage musical sheets.
